cli/injector/main.go

```diff
@@ -271,18 +274,9 @@ func initExitSignalsHandler() {
 	signal.Notify(signals, syscall.SIGINT, syscall.SIGTERM)
 }
 
-// injectAndWait injects the disruption with the configured injector and waits
-// for an exit signal to be sent
-func injectAndWait(cmd *cobra.Command, args []string) {
-	// early exit if an injector configuration failed to be generated during initialization
-	if !readyToInject {
-		log.Error("an injector could not be configured successfully during initialization, aborting the injection now")
-
-		return
-	}
-
-	log.Infow("injecting the disruption", "kind", cmd.Name())
-
+// inject inject all the disruptions using the list of injectors
+// returns true if injection succeeded, false otherwise
```

Comment:
This comment is backwards, we return true if cleanup had an error, false otherwise. Would it be better to return an error here (or a multi error), rather than a bool?